demo/app-root.ts

```diff
@@ -1,19 +1,105 @@
-import { html, css, LitElement, customElement } from 'lit-element';
-import '../src/your-webcomponent';
+import { html, css, LitElement, customElement, query } from 'lit-element';
+import '../src/iaux-underlined-tab-bar';
+import type { UnderlinedTabBar } from '../src/iaux-underlined-tab-bar';
 
 @customElement('app-root')
 export class AppRoot extends LitElement {
+  @query('iaux-underlined-tab-bar') tabBar!: UnderlinedTabBar;
+
+  private itemClicked(e: CustomEvent) {
+    console.log(e);
+    console.log(e.detail.index);
+  }
+
+  private setEntries() {
+    this.tabBar.entries = [
+      { displayName: 'CHANGE' },
+      { displayName: 'TITLE' },
+      { displayName: 'OPTIONS' },
+      { displayName: 'ACTIVE' },
+    ];
+  }
+
+  private loadControl() {
+    console.log(this.tabBar.isLoading);
+    this.tabBar.isLoading = !this.tabBar.isLoading;
+  }
+
+  private setRandomIndex() {
+    this.tabBar.selectedIndex = Math.floor(
+      Math.random() * this.tabBar.entries.length
+    );
+  }
+
+  private underlineWideInc() {
```

Comment:
another suggestion: name this `increaseUnderlineWidth()` to have an active verb in name and also have symmetry with the `increaseMinAnimationDuration()` methods